grails-app/views/index.gsp

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
```

Comment:
No views are needed. They are in the pluginExcludes array in AirbrakeGrailsPlugin class.